src/components/Tutorial/__tests__/Tutorial.ts

```diff
@@ -0,0 +1,260 @@
+  })
+
+  describe('step autoexpand', async () => {
+    it('we learn about automatic thought hiding', async () => {
+      await user.click(screen.getByText('Next'))
+      expect(screen.getByText(/thoughts are automatically hidden when you click away/)).toBeInTheDocument()
+    })
+
+    it('we hide nested thoughts by clicking away', async () => {
+      /* thoughts:
+      - first
+      - second
+      - third
+        - child of third
+      */
+      await user.click(screen.getByText('second'))
+      expect(screen.getByText(/Notice that "child of third" is hidden now/)).toBeInTheDocument()
+    })
+
+    it('we reveal hidden thoughts by clicking parent', async () => {
+      await user.click(screen.getAllByText('third').at(-1)!)
+      expect(screen.getByText('child of third')).toBeInTheDocument()
+    })
+  })
+
+  describe('step success', async () => {
+    it('we complete the first tutorial', async () => {
+      expect(screen.getByText(/Lovely\. You have completed the tutorial/)).toBeInTheDocument()
+    })
+
+    it('we can exit tutorial mode', async () => {
+      await user.click(screen.getByText('Play on my own'))
+      await act(vi.runOnlyPendingTimersAsync)
+      expect(() => screen.getByTestId('tutorial-step')).toThrow('Unable to find an element')
+    })
+
+    it('we can continue to advanced tutorial', async () => {
+      expect(screen.getByText('Learn more')).toBeInTheDocument()
+    })
+  })
+})
+
+describe('Tutorial 2', async () => {
+  beforeEach(() => createTestApp({ tutorial: true }))
+  afterEach(cleanupTestEventHandlers)
+  afterAll(cleanupTestApp)
+  it('we learn about context indicators', async () => {
+    await user.click(screen.getByText('Learn more'))
+
+    expect(screen.getByText(/shows a small number to the right of the thought, for example/)).toBeInTheDocument()
+  })
+
+  it('we choose a project type, one of three options', async () => {
+    await user.click(screen.getByText('Next'))
+    expect(screen.getByText(/For this tutorial, choose what kind of content you want to create/)).toBeInTheDocument()
+    expect(screen.getByText('To-Do List')).toBeInTheDocument()
+    expect(screen.getByText('Journal Theme')).toBeInTheDocument()
+    expect(screen.getByText('Book/Podcast Notes')).toBeInTheDocument()
+  })
+
+  it('we start creating a to-do list', async () => {
+    await user.click(screen.getAllByText('To-Do List').at(-1)!)
+    expect(screen.getByText(/Excellent choice\. Now create a new thought with the text “Home”/)).toBeInTheDocument()
+  })
+
+  describe('step context 1 - create a "Home" to-do list', () => {
+    it('we create a Home thought', async () => {
+      await user.keyboard('{Enter}')
+      await user.type(lastEmptyThought(), 'Home')
+      await act(vi.runOnlyPendingTimersAsync)
+      expect(screen.getByText(/Add a thought with the text "To Do"/)).toBeInTheDocument()
+    })
+
+    it('we add a Home->To Do subthought', async () => {
```

Comment:
Please use `Home/To Do` or `Home • To Do` to indicate subthought relationships. That will be a bit more consistent with our internal documentation, since `->` is not used.

Likewise below.